### PR TITLE
Fix typos and url in experimental

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
1. Fix typos 
2. Modify the url which isn't exist

**- How I did it**

**- How to verify it**

**- Description for the changelog**

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
The experimental/README.md itself is totally outdated; 
IMO we no longer need to mention `experimental.docker.com`, and we just need to mention the `--experimental` runtime flag.

https://github.com/docker/docker/issues/26713
https://github.com/docker/docker/pull/27223

PTAL @mlaventure @icecrime WDYT?

**Comment by AkihiroSuda** [comment]
opened https://github.com/docker/docker/pull/28160 for experimental/README.md

**Review by vdemeester — DISMISSED** [review]
LGTM 🐸

File: experimental/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ refined or entirely removed.
 	>command fails for the Docker repo during installation. To work around this,
 	>add the key directly using the following:
 	>
-	>       $ curl -sSL https://experimental.docker.com/gpg | sudo apt-key add -
+	>   $ curl -sSL https://experimental.docker.com/gpg | sudo apt-key add -
```
Comment (thaJeztah) [inline experimental/README.md]: This should not be changed, because it no longer shows as a code block

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks @yuexiao-wang 

Actually, this document may need more changes, because the experimental features are now enabled at runtime through a `--experimental` flag on `dockerd`, see https://github.com/docker/docker/pull/27223

Let me /cc @andrewhsu here as well

**Comment by thaJeztah** [comment]
I see @AkihiroSuda mentioned that as well, and has an open PR to change that. I think we should use https://github.com/docker/docker/pull/28160 instead. Let me close this PR, but thanks again for contributing @yuexiao-wang

File: experimental/plugins_graphdriver.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ being started.
 
 # Write a graph driver plugin
 
-See the [plugin documentation](/docs/extend/plugins.md) for detailed information
+See the [plugin documentation](/docs/extend/index.md) for detailed information
```
Comment (AkihiroSuda) [inline experimental/plugins_graphdriver.md]: You can open a PR for this line

**Pull request closed without merging**